app/views/stream/_about.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container about">
+  <section class="page-header">
+    <h1>Activity streams</h1>
+  </section>
+
+  <section class="main">
+    <article class="article-block">
+      <div class="article-contents">
+        <h4 class="header">How it works</h4>
+        <p> Activity Streams show you how other people approach to the problems. </p>
+        <p> When you have submitted a solution to at least one exercise, then the stream starts getting filled with the solutions that other people have submitted to the same exercise.</p>
+        <p> It will differentiate your submissions from the others by <a href="http://exercism.io/languages">langauge track</a> </p>
+        <p> You will be able to view and comment on others' submissions <p>
```

Comment:
Also, let's ditch the extra space inside the paragraph tags. They'll be rendered, I think.